### PR TITLE
For PR-940

### PR DESCRIPTION
**Pull request by yrong**

### CODE REVIEW

File: parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs [hunk 1]
```diff
@@ -677,6 +681,17 @@ parameter_types! {
...
+
+	// Multiplier based on gas report as follows,assume gas cost for a no-op operation is 1000
+	// | createAgent                                     | 839             | 184709 | 237187 | 237187 | 9       |
+	// | createChannel                                   | 399             | 31023  | 2829   | 75402  | 5       |
+	// | updateChannel                                   | 817             | 15121  | 3552   | 36762  | 5		 |
+	// | transferNativeFromAgent                         | 770             | 21730  | 21730  | 42691  | 2       |
+	pub const CreateAgentMultiplier: u128 = 237;
+	pub const CreateChannelMultiplier: u128 = 75;
+	pub const UpdateChannelMultiplier: u128 = 37;
+	pub const TransferNativeFromAgentMultiplier: u128 = 43;
```
Comment (alistair-singh) [inline parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs]: This multipliers look unused.

Comment (yrong) [reply parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs]: https://github.com/Snowfork/cumulus/pull/63/commits/1a1ab19a6e0d4449211ee24897ee7211b72cf9a1

File: parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs [hunk 2]
```diff
@@ -686,11 +689,14 @@ impl snowbridge_control::Config for Runtime {
 	type MessageHasher = BlakeTwo256;
 	type WeightInfo = weights::snowbridge_control::WeightInfo<Runtime>;
 	type MaxUpgradeDataSize = MaxUpgradeDataSize;
-	type AgentHashedDescription = AgentHashedDescription;
+	type AgentIdOf = AgentIdOf;
 	type UniversalLocation = UniversalLocation;
 	type RelayLocation = RelayLocation;
 	type AgentOrigin = EnsureXcm<Everything>;
 	type ChannelOrigin = EnsureXcm<AllowSiblingsOnly>;
+	type LocalPalletId = TreasuryPalletId;
```
Comment (yrong) [inline parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs]: Use Treasury pallet id.

Comment (claravanstaden) [reply parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs]: Shouldn't we name this type `TreasuryPalletId` for clarity?

**Pull request closed without merging**